README.md

```diff
@@ -1,2 +1,31 @@
 # community
+
 Salt Extensions Community Repository for Meeting Notes, Docs, etc.
+
+## Salt Extensions working group
+
+The Salt Extensions working group is the place to go if you would like to help
+create, improve, and maintain a Salt extension. We'll collaborate as peers to
+help ensure your Salt extensions well-designed and easy to use. It's a great
+opportunity to network while also skilling up on Salt, Python, and module
+documentation.
+
+Come talk to the Salt documentation working group by joining the
+`#salt-extensions` channel on the
+[Salt Community Slack](https://saltstackcommunity.slack.com).
+
+The current working group captains are
+[Nicholas Hughes](https://github.com/nicholasmhughes) and
+[Alyssa Rock](https://github.com/barbaricyawps).
+
+
+## Join the Salt Extensions working group
+
+To join the Salt Extensions working group:
+
+* Check the
+[community calendar](https://saltproject.io/calendar/) for meeting times and
+Zoom links.
+* Join the `#salt-extensions` channel on the
+[Salt Community Slack](https://saltstackcommunity.slack.com).
+* See the `meeting-notes` folder to read past working group meeting notes.
```

Comment:
**issue (blocking):** the folder name changed because of some jerk's nitpicks 😉

```suggestion
* See the `working_group` folder to read past working group meeting notes.
```